### PR TITLE
Added a Parent Edit tab

### PR DESCRIPTION
**Pull request by randvoid**

### CODE REVIEW

**Comment by randvoid** [comment]
Hi,

Any news for this?

File: web/viperweb/views.py [hunk 1]
```diff
@@ -776,3 +776,69 @@ def post(self, request, *args, **kwargs):
...
+
+        return redirect(reverse("file-view", kwargs={"project": project, "sha256": child}))
+
+
+class DeleteParentView(LoginRequiredMixin, TemplateView):
+    def get(self, request, *args, **kwargs):
+        return HttpResponse('This is a POST only view')
+
+    def post(self, request, *args, **kwargs):
+        print("IN FUCKING DELETE PARENT VIEW")
```
Comment (frennkie) [inline web/viperweb/views.py:819]: same here

File: web/viperweb/views.py [hunk 2]
```diff
@@ -776,3 +776,69 @@ def post(self, request, *args, **kwargs):
...
                                                    'searched_value': value,
                                                    'projects': get_project_list()})
+
+
+class AddParentView(LoginRequiredMixin, TemplateView):
+    def get(self, request, *args, **kwargs):
+        return HttpResponse('This is a POST only view')
+
+    def post(self, request, *args, **kwargs):
+        print("IN FUCKING ADD PARENT VIEW")
```
Comment (frennkie) [inline web/viperweb/views.py:786]: I guess this is not supposed to be merged, right?

**Review by frennkie — CHANGES_REQUESTED** [review]

**Comment by frennkie** [comment]
Can you check the conflicts and remove the f***ing lines.. :-D

**Comment by botherder** [comment]
Closing this for inactivity.

**Pull request closed without merging**